authority-portal-backend/gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@ java = "17"
 kotlin = "1.9.22"
 npmVersion = "8.15.0"
 
-sovity-edcCe = "7.5.0"
+sovity-edcCe = "0.0.1-SNAPSHOT"
```

Comment:
fix the release guide to contain a section about choosing a compatible EDC CE version, setting it here, and including the compatible link (release tag-pinned docs link) to the deployment guide of the crawler + working through the deployment notes for the crawler and including them in the deployment migration notes

Reply:
Will be addressed in #232 with the rest of the documentation changes